sshspawner/sshspawner.py

```diff
@@ -3,29 +3,35 @@
 import shlex
 from textwrap import dedent
 import warnings
+import random
 
-from traitlets import Bool, Unicode, Integer
+from traitlets import Bool, Unicode, Integer, List, observe
 
 from jupyterhub.spawner import Spawner
 
 
 class SSHSpawner(Spawner):
-    
+
+    # http://traitlets.readthedocs.io/en/stable/migration.html#separation-of-metadata-and-keyword-arguments-in-traittype-contructors
+    # config is an unrecognized keyword
+
+    nodes = List(trait=Unicode(), default_value=['Cori19','Cori20'],
+            help="Possible remote hosts from which to choose remote_host.").tag(config=True)
+
+    # Removed 'config=True' tag.
+    # Any user configureation of remote_host is redundant.
+    # The spawner now chooses the value of remote_host.
     remote_host = Unicode("remote_host",
-            help="SSH remote host to spawn sessions on",
-            config=True)
+            help="SSH remote host to spawn sessions on")
 
     remote_port = Unicode("22",
-            help="SSH remote port number",
-            config=True)
+            help="SSH remote port number").tag(config=True)
```

Comment:
As of traitlets 4.3.2 passing `config=True` in traitlets constructors is no longer deprecated (see https://github.com/ipython/traitlets/pull/376).  Can you revert these to pass `config=True`?  I realize this may not be consistently applied across all the custom spawners we have but we should move in that direction.